CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ endif()
 # Main project
 project(
   YARP
-  VERSION 3.8.1
+  VERSION 3.8.2
```

Comment:
This should be never modified by a PR. It is handled by the release script

Reply:
Done in https://github.com/robotology/yarp/pull/3010/commits/ad556c7cfb072dd58387086edc9791f0137e72f1